setup.py

```diff
@@ -63,7 +63,7 @@ def get_version():
     'django >=1.6, <1.10',
     'requests >=2.2, <2.13',
     'pyshp ==1.2',
```

Comment:
1.2.3 works for sure, I am pretty lower versions do not work, not sure about higher versions.